charts/gardener/provider-local/templates/rbac.yaml

```
@@ -129,7 +129,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ include "name" . }}
+  name: {{ .Release.Namespace }}
```

This looks like the hack from the Hackathon 😉 
```suggestion
  name: {{ include "name" . }}-{{ .Release.Namespace }}
```
Maybe at least?

---

I need to rethink what I consider a hack 😅. The `name` template it really is `gardener-extension-provider-local` so the end result would be something like `gardener-extension-provider-local-extension-provider-local-<hash>`